client/src/components/posting-page/JobPosting.js

```diff
@@ -110,7 +110,10 @@ export class JobPosting extends Component {
           <AdminButtons id={jobPosting.id} />
         }
         <h2 className='job-posting__title'>{jobPosting.title}</h2>
-        {jobPosting.isHidden && <HiddenNotification jobPosting={jobPosting} />}
+        {jobPosting.isHidden &&
+          <HiddenNotification showFrom={jobPosting.showFrom} showTo={jobPosting.showTo} />
+        }
+        {errorMessage && <ErrorMessage errorMessage={errorMessage} />}
```

Comment:
Is this ErrorMessage necessary anymore? This throws error for me, and I don't think this is needed as we started to use the Snackbar.

Reply:
Yeah that's true, it's not needed. It wasn't added by me, but I'll delete it anyway